themes/bootstrap3/templates/_ui/components/search-offcanvas-show.phtml

```diff
@@ -0,0 +1,10 @@
+<?php
+    if (null === ($side = $this->config()->offcanvasSide())) {
+        return '';
+    }
+?>
+<a class="search-filter-toggle icon-link" href="#search-sidebar" data-toggle="vufind-offcanvas" title="<?=$this->transEscAttr('sidebar_expand') ?>">
```

Comment:
The link text is also different. I thought about a common template for them, but decided to keep the templates independent. I can add a common template they both use, if you think that's clearer.

Reply:
In any case, if we keep them separate, it's easier for customization. But of course harder for maintenance.

Reply:
Do you think there's a likely use case where somebody customizes one of these but not the other? It seems more likely that the user would want the same button behavior in both cases. Making the template more generic not only helps with that but also reduces the amount of redundant hard-coded strings in the template code (since the link is generated two different ways, and the text is currently repeated in both cases).

If somebody really did want to customize this differently in each situation, it would be possible to add a conditional based on the href.

All of these things make me lean toward a single template with parameters instead of two separate templates. But if you have a really strong use case for two templates, I can be persuaded otherwise. :-)